guides/v2.1/cloud/release-notes/cloud-tools.md

```diff
@@ -10,16 +10,23 @@ functional_areas:
   - Setup
   - Configuration
 ---
+The ece-tools package, also known as _ece-tools_, is compatible with Magento version 2.1.4 and later to provide a rich set of features you can use to manage your {{site.data.var.ee}} project.  It contains scripts and {{site.data.var.ece}} commands designed to help manage your code and automatically build and deploy your Cloud projects.
 
-{% include cloud/ece-tools.md %}
+You can list the available ece-tools commands using:
+
+```bash
+php ./vendor/bin/ece-tools list
+```
+
+The following updates describe the latest improvements to the ece-tools package, which update as needed through patching and product upgrades managed by the `magento-cloud-metapackage`. The ece-tools package adheres to the version sequence:  `200<major>.<minor>.<patch>`.
 
 ## v2002.0.12
 
-{% include note.html type="info" content="The Cloud tools version 2002.0.12 now supports Magento 2.1.14. " %}
+{% include note.html type="info" content="The ece-tools version 2002.0.12 now supports Magento 2.1.14. " %}
 
 #### New Features
 
--  <!-- MAGECLOUD-2250 -->**Docker Compose for Cloud**—Added a new command—`docker:build`—to generate a [Docker Compose]({{ page.baseurl }}/cloud/reference/docker-config.html) configuration rom the Cloud tools `ece-tools` repository.
+-  <!-- MAGECLOUD-2250 -->**Docker Compose for Cloud**—Added a new command—`docker:build`—to generate a [Docker Compose]({{ page.baseurl }}/cloud/reference/docker-config.html) configuration from the Cloud `ece-tools` repository.
```

Comment:
Shouldn't we be using the fully qualified name for the product via a Jekyll variable?

>"...configuration from the ~Cloud~ {{site.data.var.ece}} `ece-tools` repository."